libibverbs/libibverbs.map.in

```diff
@@ -162,6 +162,7 @@ IBVERBS_1.13 {
 IBVERBS_1.14 {
 	global:
 		ibv_query_qp_data_in_order;
+		ibv_wr_opcode_str;
```

Comment:
This is internal function and needs to be in IBVERBS_PRIVATE_@IBVERBS_PABI_VERSION@ section together with bump of IBVERBS_PABI_VERSION.